server/schedule/operator/step.go

```diff
@@ -29,20 +31,38 @@ import (
 // OpStep describes the basic scheduling steps that can not be subdivided.
 type OpStep interface {
 	fmt.Stringer
-	ConfVerChanged(region *core.RegionInfo) bool
+	ConfVerChanged(region *core.RegionInfo) uint64
 	IsFinish(region *core.RegionInfo) bool
 	CheckSafety(region *core.RegionInfo) error
 	Influence(opInfluence OpInfluence, region *core.RegionInfo)
 }
 
+var (
+	_ OpStep = (*TransferLeader)(nil)
+
+	_ OpStep = (*AddPeer)(nil)
+	_ OpStep = (*AddLightPeer)(nil)
+	_ OpStep = (*RemovePeer)(nil)
+	_ OpStep = (*AddLearner)(nil)
+	_ OpStep = (*AddLightLearner)(nil)
+
+	_ OpStep = (*PromoteLearner)(nil)
+	_ OpStep = (*DemoteFollower)(nil)
+	_ OpStep = (*ChangePeerV2Enter)(nil)
+	_ OpStep = (*ChangePeerV2Leave)(nil)
+
+	_ OpStep = (*MergeRegion)(nil)
+	_ OpStep = (*SplitRegion)(nil)
+)
```

Comment:
What's these codes for?

Reply:
for [Go Style Guide](https://github.com/uber-go/guide/blob/master/style.md#verify-interface-compliance)